AutomatedTesting/Assets/TestAnim/scene_export_actor.py

```diff
@@ -208,7 +208,6 @@ def on_update_manifest(args):
 
     global sceneJobHandler
     sceneJobHandler.disconnect()
-    sceneJobHandler = None
```

Comment:
Suggestion: It would be nice to add a comment in these Python files explaining that the sceneJobHandler cannot be deallocated while inside a callback or else invalid memory accesses will occur. That way it will be more obvious that "sceneJobHandler = None" shouldn't get added back in at some point.

Reply:
I'll put it in after each disconnect.